doc/api/v8.md

```diff
@@ -131,13 +131,24 @@ Returns an object with the following properties:
 * `malloced_memory` {number}
 * `peak_malloced_memory` {number}
 * `does_zap_garbage` {number}
+* `number_of_native_contexts` {number}
+* `number_of_detached_contexts` {number}
+
 
 `does_zap_garbage` is a 0/1 boolean, which signifies whether the
 `--zap_code_space` option is enabled or not. This makes V8 overwrite heap
 garbage with a bit pattern. The RSS footprint (resident memory set) gets bigger
 because it continuously touches all heap pages and that makes them less likely
 to get swapped out by the operating system.
 
+`number_of_native_contexts` The value of native_context is the number of the
+top-level contexts created. Increase of this number over time indicates
```

Comment:
```suggestion
top-level contexts currently active. Increase of this number over time indicates
```

(Because otherwise it’s a big ambiguous, it sounds like it might count all contexts ever created, not just the ones still alive)